src/microbe_stage/PatchMap.cs

```diff
@@ -215,8 +215,10 @@ public List<Species> RemoveExtinctSpecies(bool playerCantGoExtinct = false)
 
         foreach (var patch in Patches)
         {
+            // We remove a species as extinct when its population value is not strictly positive,
+            // unless it the species of the player and the player can't go extinct.
```

Comment:
There's a minor typo but this part could just be reworded for ease of reading.
```suggestion
            // unless it's the player's species and the player can't go extinct.
```